packages/flame_test/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## [next]
+ - Adding `flameTest` and `flameWidgetTest`
```

Comment:
nit: `Add` (to be consistent with the other entry below)

Reply:
```suggestion
 - Add `flameTest` and `flameWidgetTest`
```